katsdpcontroller/test/test_sensor_proxy.py

```diff
@@ -94,6 +95,8 @@ def _check_sensors(self) -> None:
         """Compare the upstream sensors against the mirror"""
         for sensor in self.server.sensors.values():
             qualname = 'prefix-' + sensor.name
+            if sensor.name == 'bytes-sensor':
+                qualname = 'custom-bytes-sensor'
```

Comment:
Any particular reason this one needed to be updated?

Reply:
Because of the `renames` passed at line 88. I picked one arbitrarily to give the renaming treatment.